code/src/App.js

```diff
@@ -1,9 +1,58 @@
-import React from 'react'
```

Comment:
HI Irina! It's a pleasure to review your neat and functional code. ✨😊

I already checked the mobile, laptop and desktop display of your app and I can say it's flawless. everything works and looks tidy and clear. So let's move on to the details 🔎.  Good job naming the components, by the way!😄